Assets/Scripts/NarrationItem.cs

```diff
@@ -1,23 +1,38 @@
 using System;
-
+using System.Collections.Generic;
+using System.ComponentModel;
 using UnityEngine;
-[Serializable]
-public class NarrationItem {
-    [SerializeField]
-    private string id;
-    [SerializeField]
-    private string story;
-    [SerializeField]
-    private string section;
-    [SerializeField]
-    private string triggered;
-    [SerializeField]
-    private string character;
-    [SerializeField]
-    private string line;
-    [SerializeField]
-    private string portrait;
-    [SerializeField]
-    private string context;
+using UnityEngine.UI;
+
+[CreateAssetMenu(fileName = "Narrative", menuName = "Narrative/Narrative Item")]
+public class NarrationItem : ScriptableObject{
+    [Tooltip("Day the narration occurs")]
+    public Day                 day;
+    [Tooltip("Character who is speaking")]
+    public Character character;
+    [Tooltip("Sounds that play in order from the beginning of the narration")]
+    public List<AudioClip> sounds;
+    [TextArea,Tooltip("Text that is spoken by the character")]
+    public string line;
+    [Tooltip("Next Narrative item. 1 or more")]
+    public List<NextNarrative> next;
+    [Tooltip("Image to be displayed behind the characters")]
+    public Image background;
 
 }
+
+[Serializable]
+public class NextNarrative {
+    [SerializeField, Tooltip("What triggers the next narration.")]
+    public Button        button;
+    [SerializeField, Tooltip("The next narration to occur")]
+    public NarrationItem narrativeItem;
+}
+
+public enum Day {
```

Comment:
This is what I meant by "It could be done a million different ways", this enum is another way to do what I already have made. I just feel like its too overkill, and now there is like redundant code, because 2 game objects are checking for the same thing, I advice that this be made through the a manager.

Reply:
I didn't see this functionality in your code. What file is doing it?

Reply:
You were commenting on it asking about the pre, mid and post things.

Reply:
I am not sure how that implements the days.